content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -93,6 +99,12 @@ to manage the lifecycle of Istio.
     $ helm install install/kubernetes/helm/istio --name istio --namespace istio-system
     {{< /text >}}
 
+    > If you want to enable global mutual TLS, set `global.mtls.enabled` to `true`:
```

Comment:
Same here, no need for the >

There's another preexisting instance of the same pattern in this file which should be removed.

Reply:
Done.